introduction.md

```diff
@@ -0,0 +1 @@
+Hi! My name is Victor.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.